index.html

```diff
@@ -3703,35 +3709,39 @@ <h1>Resolution</h1>
 document</a> in at least one conformant <a>representation</a>.
     </p>
 
-    <section>
+    <section data-dfn-for="Resolver">
         <h2>DID Resolution</h2>
         <p>
-The <a>DID resolution</a> functions resolve a <a>DID</a> into a <a>DID
-document</a> by using the "Read" operation of the applicable <a>DID method</a>.
-(See <a href="#read-verify"></a>.) The details of how this process is
-accomplished are outside the scope of this specification, but all conformant
-implementations implement two functions which have the following abstract
+The <a>DID resolution</a> function resolves a <a>DID</a> into a <a>DID
```

Comment:
```suggestion
The <a>DID resolution</a> functions resolves a <a>DID</a> into a <a>DID
```

Reply:
If the `functions` are plural, the `resolve` should be singular.
If the `function` is singular, the `resolves` should be plural.

Reply:
Fixed via a reword.